### ISSUE TITLE
AWS SES Callback returning 500 for 2.15.0 - worked fine in previous versions

### ISSUE THREAD

**Issue by J-Light** [mautic/mautic#7217]
## Bug Description
We have been using AWS SES for our email service and SNS for managing bounces as per documentation. The setup was work and routinely verified.

Once we updated to 2.15.0 we noticed we were not getting any bounces. Our tests with AWS Simulator resulted in the bounces failing to be handled correctly. It appears that the AWS callback is not working as expected. 

Accessing the following URL results in a 500 Internal Server Error
https://our.url/mailer/amazon/callback

| Q   | A
| --- | ---
| Mautic version | 2.15.0
| PHP version | PHP 7.0.32-0ubuntu0.16.04.1
| Browser | Chrome 

### Steps to reproduce
1. Setup emails using Amazon SES (dont have to move out of Amazon SES sandbox ) 
2. Setup bounce handling using Amazons SNS from Mautic Documentation
2. Send emails to  [AWS Mailbox Simulator](https://docs.aws.amazon.com/ses/latest/DeveloperGuide/mailbox-simulator.html)
3. See that bounces and complaints fail. 
 
### Log errors
```html
[2019-02-05 15:03:07] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Type error: Argument 1 passed to Mautic\EmailBundle\Swiftmailer\Transport\AmazonTransport::processCallbackRequest() must be an instance of Symfony\Component\HttpFoundation\Request, null given, called in /html/app/bundles/EmailBundle/EventListener/EmailSubscriber.php on line 172" at /html/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonTransport.php line 97 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Type error: Argument 1 passed to Mautic\\EmailBundle\\Swiftmailer\\Transport\\AmazonTransport::processCallbackRequest() must be an instance of Symfony\\Component\\HttpFoundation\\Request, null given, called in /html/app/bundles/EmailBundle/EventListener/EmailSubscriber.php on line 172 at /html/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonTransport.php:97)"} []
[2019-02-05 15:03:31] mautic.NOTICE: PHP Notice - Undefined variable: request - in file /html/app/bundles/EmailBundle/Event/TransportWebhookEvent.php - at line 56 [] []
[2019-02-05 15:03:31] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Type error: Argument 1 passed to Mautic\EmailBundle\Swiftmailer\Transport\AmazonTransport::processCallbackRequest() must be an instance of Symfony\Component\HttpFoundation\Request, null given, called in /html/app/bundles/EmailBundle/EventListener/EmailSubscriber.php on line 172" at /html/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonTransport.php line 97 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Type error: Argument 1 passed to Mautic\\EmailBundle\\Swiftmailer\\Transport\\AmazonTransport::processCallbackRequest() must be an instance of Symfony\\Component\\HttpFoundation\\Request, null given, called in /html/app/bundles/EmailBundle/EventListener/EmailSubscriber.php on line 172 at /html/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonTransport.php:97)"} []
[2019-02-05 15:05:35] mautic.NOTICE: PHP Notice - Undefined variable: request - in file /html/app/bundles/EmailBundle/Event/TransportWebhookEvent.php - at line 56 [] []
[2019-02-05 15:05:35] mautic.CRITICAL: Uncaught PHP Exception Symfony\Component\Debug\Exception\FatalThrowableError: "Type error: Argument 1 passed to Mautic\EmailBundle\Swiftmailer\Transport\AmazonTransport::processCallbackRequest() must be an instance of Symfony\Component\HttpFoundation\Request, null given, called in /html/app/bundles/EmailBundle/EventListener/EmailSubscriber.php on line 172" at /html/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonTransport.php line 97 {"exception":"[object] (Symfony\\Component\\Debug\\Exception\\FatalThrowableError(code: 0): Type error: Argument 1 passed to Mautic\\EmailBundle\\Swiftmailer\\Transport\\AmazonTransport::processCallbackRequest() must be an instance of Symfony\\Component\\HttpFoundation\\Request, null given, called in /html/app/bundles/EmailBundle/EventListener/EmailSubscriber.php on line 172 at /html/app/bundles/EmailBundle/Swiftmailer/Transport/AmazonTransport.php:97)"} []
```

[//]: # ( Invisible comment:
Please check for related errors in the latest log file in [mautic root]/app/log/ and/or the web server's logs and post them here. Be sure to remove sensitive information if applicable. )

**Comment by maltonite** [comment]
There is a fix to this; see pr #7015

**Comment by maltonite** [comment]
Duplicate of #7199

**Comment by colourfulowl** [comment]
The fix highlighted in #7199 resolved my error. I suggest doing the same :)
